contribs/gmf/less/mobile-nav.less

```diff
@@ -44,6 +44,30 @@ main {
   }
 }
 
+.search-overlay {
+  /* shadow layer visible when search is active */
+  position: absolute;
+  z-index: @above-menus-index;
+  height: 100%;
+  width: 100%;
```

Comment:
Does it make a difference with `position: absolute`? Or what advantages does it have?